src/factory/constants.ts

```diff
@@ -51,14 +52,15 @@ export const CONTRACT_ADDRESSES: Record<
     ...MasterCopyAddresses,
     [KnownContracts.OPTIMISTIC_GOVERNOR]:
       "0x1340229DCF6e0bed7D9c2356929987C2A720F836",
+    [KnownContracts.OZ_GOVERNOR]: "0x119bAebEDCF2A32dFAc3868Cd2F446db744ab675",
   },
   56: { ...MasterCopyAddresses },
   100: { ...MasterCopyAddresses },
   137: {
     ...MasterCopyAddresses,
     [KnownContracts.TELLOR]: "0xEAB27A2Dc46431B96126f20bFC3197eD8247ed79",
     [KnownContracts.OPTIMISTIC_GOVERNOR]:
-      "0x923b1AfF7D67507A5Bdf528bD3086456FEba10cB",
+      "0x7E67f2D27B0193980F49DcCed96fE3Ac1Ce44641",
```

Comment:
Why does the Optimistic Governor address change?

Reply:
Ops! Was suppose to go in the OZ Governor 😅 

Thank you